src/web/app/components/response-time-series-chart/response-time-series-chart.component.ts

```diff
@@ -0,0 +1,129 @@
+import { Component, Input, OnInit } from '@angular/core';
+import * as d3 from 'd3';
+import { ResponseTimeSeriesChartModel } from "./response-time-series-chart.model";
+import { FeedbackResponseStatsService } from "../../../services/feedback-response-stats.service";
+import { BehaviorSubject, interval } from "rxjs";
+import { switchMap, tap } from "rxjs/operators";
+import { FeedbackResponseStats } from "../../../types/api-output";
+import { ErrorMessageOutput } from "../../error-message-output";
+
+/**
+ * Response-tracking time series chart for admin user.
+ */
+@Component({
+  selector: 'tm-response-time-series-chart',
+  templateUrl: './response-time-series-chart.component.html',
+  styleUrls: ['./response-time-series-chart.component.scss']
+})
+export class ResponseTimeSeriesChartComponent implements OnInit {
+
+  @Input()
+  model: ResponseTimeSeriesChartModel = {
+    durationMinutes: 60,
+    intervalMinutes: 1
+  }
+
+  interval: BehaviorSubject<number> = new BehaviorSubject<number>(1);
+
+  constructor(private feedbackResponseStatsService: FeedbackResponseStatsService) { }
+
+  ngOnInit(): void {
+    this.interval.pipe(
+        switchMap(value => interval(value * 1000)), // in seconds (for easier testing), actual one will be in minutes
+        tap(() => this.refresh())
+    ).subscribe();
+  }
+
+  refresh() {
+    this.feedbackResponseStatsService.loadResponseStats(this.model.durationMinutes.toString(), this.model.intervalMinutes.toString())
+        .subscribe((data: FeedbackResponseStats) => {
+          console.log(data);
```

Comment:
For future reference: don't commit `console.log` or `System.out.println` lines.
iirc IDE has this awesome feature that allows you to commit lines in a file.

Reply:
added in to test if there's data coming from the backend, will remove it before it is ready to be merged